processes/recognition.md

```diff
@@ -0,0 +1,47 @@
+# Recognition
+
+Outlining the processes, tools and resources available for recognizing the work being done in the space of the Security WG security researchers.
+
+## Recognizing Security Researchers
+
+We recognize the effort, and time spent by individual security researchers to submit reports and take part in the process of triaging and communicating with different projects and maintainers in order to resolve a security issue and create a better ecosystem for Node.js.
+
+The following are tools and processes that may and should be used in order to recognize and credit this work:
+
+### Social Media 
+
+A monthly tweet will be sent out from the official `@nodejs` account on the 25th that will mention four security researchers and their contribtion based off of information that the Security WG members will provide in the following [spreadsheet](https://docs.google.com/spreadsheets/d/1yMqH9FhzmwxJuBarbqEdr2Z1Re491nt4st68HZjOHSA/edit#gid=0)
+
+**Hashtag**: #nodejssecurity
+
+**Contact person**: Zibby Keaton (@ZibbyKeaton)
+
+Example link for tweet reference: https://twitter.com/nodejs/status/1012720630964326402
+
+
+### Bug Bounty Program Leaderboard
+
+HackerOne maintains a leaderboard of active members in each program which can be queried to find members to recognize:
+* https://hackerone.com/nodejs-ecosystem/thanks
+* https://hackerone.com/nodejs/thanks
+
+### Quarterly Spotlight
+
+A Quarterly Spotlight will be a regular post from the Security WG that will list recent work from processes to vulnerabilities, in which we can feature several active hackers in our community and their disclosures.
+
+### Node.js Website TBD
+
+Official recognition through listing on the Node.js website.
+TBD, see PR: https://github.com/nodejs/website-redesign/issues/59
+
+
+### Swag TBD
+
+* Stickers
+* T-shirts
+* Books
+
+### Recognition for Significant Contribution TBD
+
+Defining a process/threshold for what constitutes a significant contribution and what it yields (recognition artifacts, joining the wg, etc)
```

Comment:
I think joining the WG would be a good way to attract new members!

Reply:
👍